Dashboard/ViewModels/StatusViewModel.cs

```diff
@@ -44,6 +44,49 @@ public StatusViewModel()
     public bool ServiceBrakeState => _dataStore.AvStatusData?.ServiceBrakeState ?? false;
     public bool RemoteEmergency => _dataStore.ResData?.ResState ?? false;
 
+    public int SatCount => _dataStore.GpsData?.SatCount ?? 0;
+
+    public double Hdop => _dataStore.GpsData?.Hdop ?? 0;
+
+    public int HdopFixAge => _dataStore.GpsData?.HdopFixAge ?? 0;
+
+    public int HVal => _dataStore.GpsData?.HVal ?? 0;
+
+    public int SatFixAge => _dataStore.GpsData?.SatFixAge ?? 0;
+
+    public int SpeedFixAge => _dataStore.GpsData?.SpeedFixAge ?? 0;
+
+    public int AltFixAge => _dataStore.GpsData?.AltFixAge ?? 0;
+
+    public double AltitudeMetres => _dataStore.GpsData?.AltitudeMetres ?? 0;
+
+    public double AltitudeKilometres => _dataStore.GpsData?.AltitudeKilometres ?? 0;
+
+    public double MetresPerSecond => _dataStore.GpsData?.MetresPerSecond ?? 0;
+
+    public double KilometresPerHour => _dataStore.GpsData?.KilometresPerHour ?? 0;
+
+    public double Latitude => _dataStore.GpsData?.Latitude ?? 0;
+
+    public double Longitude => _dataStore.GpsData?.Longitude ?? 0;
+
+    public bool ResState => _dataStore.ResData?.ResState ?? false;
+
+    public bool K2State => _dataStore.ResData?.K2State ?? false;
+
+    public bool K3State => _dataStore.ResData?.K3State ?? false;
+
+    public int ResRadioQuality => _dataStore.ResData?.ResRadioQuality ?? 0;
+
+    public int ResNodeId => _dataStore.ResData?.ResNodeId ?? 0;
+
+    public void Dispose()
+    {
+        _dataStore.Dispose();
+    }
+
+    public event Action<bool>? RESDataUpdated;
+
     private void OnAvDataChanged(object? sender, EventArgs e)
```

Comment:
This has variables from RES and GPS, it should be broken out into OnRES, OnAV and OnGPS 'DataChanged events so it isn't reliant on getting AV data events to trigger updates.